src/components/SupportedClients.js

```diff
@@ -7,9 +7,6 @@ function buildButton(type) {
     if (type === 'cli' || type === 'c') {
         return <span class="badge badge--secondary">CLI</span>
     }
-    if (type === 'android' || type === 'a') {
-        return <span class="badge badge--success">Android</span>
-    }
```

Comment:
May be worth leaving this one?
```suggestion
    if (type === 'android' || type === 'a') {
        return <span class="badge badge--success">Android</span>
    }
```